develop/webservices/WS_v1/WS_Methods_v1/GetUsers.md

```diff
@@ -17,3 +17,6 @@ Use this method to retrieve the users configured on a DMA.
 | Item           | Format           | Description                                         |
 |----------------|------------------|-----------------------------------------------------|
 | GetUsersResult | Array of [DMAUser](xref:DMAUser) | An array of DMAUser objects, consisting of the login name, full name, and email address of each user. |
```

Comment:
DMAUser was extended with `int[] GroupIds`, a list of IDs of group that user belongs to.

Reply:
I've added this info now. OK to merge?